README.md

```diff
@@ -15,3 +44,101 @@ table = Table("table-id", MetaData(bind=engine), autoload=True)
 for row in select(["*"], from_obj=table).execute().fetchall():
     print(row)
 ```
+
+**Create a table**
+```python
+from sqlalchemy import (
+    Column,
+    Integer,
+    MetaData,
+    String,
+    Table,
+    create_engine,
+)
+
+engine = create_engine(
+    "spanner:///projects/appdev-soda-spanner-staging/instances/sqlalchemy-dialect-test/databases/compliance-test"
+)
+metadata = MetaData()
+
+user = Table(
+    "users",
+    metadata,
+    Column("user_id", Integer, primary_key=True),
+    Column("user_name", String(16), nullable=False),
+)
+
+metadata.create_all(engine)
+```
+Migration
+-----------
+SQLAlchemy uses [Alembic](https://alembic.sqlalchemy.org/en/latest/#) tool to organize database migrations.
```

Comment:
Is this going to work out of the box or is there manual adjustments a user would need to make to get it working with Cloud Spanner?

Reply:
It works out of the box, as Alembic is built on top of SQLAlchemy, and inside a migration script simple SQLAlchemy commands are used. Meaning, if SQLAlchemy works fine with Spanner than Alembic should also work fine. I assume we should deal with the last data types problems in the dialect, and then we'll be able to write tests for migrations.